.github/CODEOWNERS

```diff
@@ -0,0 +1,4 @@
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners.
+
+*                                    mirceanis
```

Comment:
This seems to be invalid:

> Invalid owner on line 4

Ideally this should be a team within the MetaMask org.

Reply:
done. I set `@MetaMask/engineering` for now